.env

```diff
@@ -0,0 +1 @@
+PORT=3000
```

Comment:
I think it's better to commit a .env-example or .env.development instead of a .env

I suggest just renaming this

Reply:
As you told me, I changed the name of .env to .env.example and I added the .env into the .gitignore.